198597ee329c4c205f8852779e6a49 Mon Sep 17 00:00:00 2001
From: Thiago Brito <thiago.brito@windriver.com>
Date: Mon, 4 Oct 2021 10:04:21 -0300
Subject: [PATCH] Disabling helm3_hook

Since openstack-helm now defaults to use helmv3 hooks, this
changes disables it to maintain compatibility with the helm
version in use on StarlingX.

Signed-off-by: Thiago Brito <thiago.brito@windriver.com>
Change-Id: I2a343805be2bb20f39b2dd8cc8d2e8716961ea28
---
 barbican/values.yaml  | 2 +-
 cinder/values.yaml    | 2 +-
 glance/values.yaml    | 2 +-
 heat/values.yaml      | 2 +-
 keystone/values.yaml  | 2 +-
 neutron/values.yaml   | 2 +-
 nova/values.yaml      | 2 +-
 placement/values.yaml | 2 +-
 8 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/barbican/values.yaml b/barbican/values.yaml
index 72efd88c..44d3b76b 100644
--- a/barbican/values.yaml
+++ b/barbican/values.yaml
@@ -31,7 +31,7 @@ release_group: null

 # NOTE(philsphicas): the pre-install hook breaks upgrade for helm2
 # Set to false to upgrade using helm2
-helm3_hook: true
+helm3_hook: false

 images:
   tags:
diff --git a/cinder/values.yaml b/cinder/values.yaml
index a70ed72a..30a2e47a 100644
--- a/cinder/values.yaml
+++ b/cinder/values.yaml
@@ -1516,7 +1516,7 @@ network_policy:

 # NOTE(helm_hook): helm_hook might break for helm2 binary.
 # set helm3_hook: false when using the helm2 binary.
-helm3_hook: true
+helm3_hook: false

 manifests:
   certificates: false
diff --git a/glance/values.yaml b/glance/values.yaml
index a03a69a1..459f04bc 100644
--- a/glance/values.yaml
+++ b/glance/values.yaml
@@ -1052,7 +1052,7 @@ pod:

 # NOTE(helm_hook): helm_hook might break for helm2 binary.
 # set helm3_hook: false when using the helm2 binary.
-helm3_hook: true
+helm3_hook: false

 manifests:
   certificates: false
diff --git a/heat/values.yaml b/heat/values.yaml
index 58f786b6..9e9ea64b 100644
--- a/heat/values.yaml
+++ b/heat/values.yaml
@@ -1293,7 +1293,7 @@ network_policy:

 # NOTE(helm_hook): helm_hook might break for helm2 binary.
 # set helm3_hook: false when using the helm2 binary.
-helm3_hook: true
+helm3_hook: false

 manifests:
   certificates: false
diff --git a/keystone/values.yaml b/keystone/values.yaml
index ff4493f3..84b8fe48 100644
--- a/keystone/values.yaml
+++ b/keystone/values.yaml
@@ -31,7 +31,7 @@ release_group: null

 # NOTE(gagehugo): the pre-install hook breaks upgrade for helm2
 # Set to false to upgrade using helm2
-helm3_hook: true
+helm3_hook: false

 images:
   tags:
diff --git a/neutron/values.yaml b/neutron/values.yaml
index c72a55b5..6073c6a3 100644
--- a/neutron/values.yaml
+++ b/neutron/values.yaml
@@ -2514,7 +2514,7 @@ network_policy:
     egress:
       - {}

-helm3_hook: true
+helm3_hook: false

 manifests:
   certificates: false
diff --git a/nova/values.yaml b/nova/values.yaml
index 6b51b3d5..56f1dc9a 100644
--- a/nova/values.yaml
+++ b/nova/values.yaml
@@ -2563,7 +2563,7 @@ network_policy:

 # NOTE(helm_hook): helm_hook might break for helm2 binary.
 # set helm3_hook: false when using the helm2 binary.
-helm3_hook: true
+helm3_hook: false

 manifests:
   certificates: false
diff --git a/placement/values.yaml b/placement/values.yaml
index 57d9eb48..2811b200 100644
--- a/placement/values.yaml
+++ b/placement/values.yaml
@@ -485,7 +485,7 @@ dependencies:

 # NOTE(helm_hook): helm_hook might break for helm2 binary.
 # set helm3_hook: false when using the helm2 binary.
-helm3_hook: true
+helm3_hook: false

 manifests:
   certificates: false
-- 
2.17.1


